packages/@aws-cdk/aws-eks/lib/helm-chart/index.py

```diff
@@ -1,10 +1,7 @@
-import subprocess
-import os
 import json
```

Comment:
Any reason why this is `python` when the other is `Javascript`? I'm strongly in favor of reducing the amount of different runtimes involved in all those custom resources we maintain (increases barrier to contributions by requiring multiple languages, makes it more difficult to track security & reliability issues that need patching, ...).

Reply:
yes, I need to use the AWS CLI (from the layer) and it requires a python runtime